apis/compute/v1beta1/referencers.go

```diff
@@ -61,7 +62,7 @@ func (mg *GlobalAddress) ResolveReferences(ctx context.Context, c client.Reader)
 		Extract:      NetworkURL(),
 	})
 	if err != nil {
-		return err
+		return errors.Wrap(err, "spec.network")
```

Comment:
Would it be worth to have errors for ref and selectors separately? Like `spec.forProvider.networkRef could not be resolved` and `spec.forProvider.networkSelector could not select` etc.

Reply:
It seems a little like diminishing returns to me, but more context is usually better. I'd tackle that inside https://github.com/crossplane/crossplane-runtime/blob/dc04a11/pkg/reference/reference.go#L182; i.e. update the existing error wrap strings such that it's obvious whether the error happened while selecting or resolving a reference. It's _possible_ to determine where an error is occurring at the moment (because the error is wrapped with "cannot get" for references and "cannot list" for selectors), but pretty opaque.

Reply:
Followed up with https://github.com/crossplane/crossplane-runtime/pull/213